bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/scoping/ScriptImplicitlyImportedTypes.java

```diff
@@ -90,15 +82,11 @@
     @Override
     protected List<Class<?>> getStaticImportClasses() {
         List<Class<?>> result = super.getStaticImportClasses();
+        result.add(ScriptExecution.class);
         result.add(BusEvent.class);
         result.add(Exec.class);
         result.add(HTTP.class);
         result.add(Ping.class);
-        result.add(ScriptExecution.class);
-        result.add(LogAction.class);
-        result.add(Audio.class);
-        result.add(Voice.class);
-        result.add(Things.class);
```

Comment:
Are the removals in this class because the rule engine already gets this information provided by `ActionService`s?

Reply:
Yes, exactly. This seemed duplicated and I checked that it works without it.

Reply:
Than we can safely merge this. :+1: